site-src/geps/gep-709.md

```diff
@@ -207,19 +217,21 @@ effective safeguards are in place.
 
 ## ForwardTo
 
-To enable cross-namespace forwarding, we'll need to add a `namespace` field to
-the ForwardTo BackendRef struct.
+To enable cross-namespace forwarding, we'll need to add an optional `namespace`
+field to the ForwardTo BackendRef struct.
 
 ```go
 type BackendRef struct {
     // ...
 
-    // Namespace is the namespace of the referrent.
+    // Namespace is the namespace of the backend. When unspecified, the local
+    // namespace is inferred.
     //
     // Support: Core
     //
     // +kubebuilder:validation:MinLength=1
     // +kubebuilder:validation:MaxLength=253
+    // +optional
```

Comment:
Should the type also change to `string*`?

Reply:
Updated, thanks!